### ISSUE TITLE
Download button doesn't do anything, some way to toggle Action frame visibility with Download/Embed/About buttons?

### ISSUE THREAD

**Issue by mgancarzdsi** [tunapanda/h5p-standalone#5]
This is an enhancement request, as I can't seem to figure out how to disable the Download/Embed/About buttons or Action frame.  It also looks like the Download button doesn't actually work to do anything, though it's shown by default.

In the Wordpress version, you can toggle them with the following options, but I don't see a way to control these in the standalone version. Could just be a setting in a json file somewhere?
 Display action bar and frame
 Download button
 Embed button
 Copyright button

**Comment by Jakeii** [comment]
Good points, I'm not sure how this is done, in my own projects I've been hiding with CSS

**Comment by mgancarzdsi** [comment]
This code looks to be responsible for it, I poked it a bit and tried removing it, but didn't get the action bar removed as expected:
https://github.com/tunapanda/h5p-standalone/blob/master/dist/js/h5p-standalone-main.js#L10509-L10551

I also tried messing around with the constant values here: https://github.com/tunapanda/h5p-standalone/blob/master/dist/js/h5p-standalone-main.js#L10416-L10428

No joy still. I guess CSS hiding will work fine though

**Comment by mrjarbenne** [comment]
Could they be hidden by default directly in this repo, particularly because the buttons don't actually work. It seems to me I was able to hide the buttons by tweaking the CSS in dist/styles/hp5.css by changing these to display: none;

https://github.com/tunapanda/h5p-standalone/blob/566c1e9408f67afe84570e1fc74c3bb5455f09fa/dist/styles/h5p.css#L198-#L220

I've tried to sort out where in the unzipped H5P directory these might be being called. Given that the WordPress plugin allows you to turn them on and off it must be somewhere, but once you turn off the "Download" option, I can't sort out how to get a downloaded copy of the H5P archive in order to compare the two.
